clip.py

```diff
@@ -74,11 +72,12 @@ def __init__(self, audio_file=None, name='',
         self.state = Clip.STOP
         self.audio_file = audio_file
         self.last_offset = 0
+        self.route_out = route_out
+        self.mute_group = mute_group
 
 
 class Song():
-
-    def __init__(self, width, height):
+    def __init__(self, width, height, outputs):
```

Comment:
here you can remove `outputs` var in constructor, outputs can be compute with clips